src/navigator/views/layout-view.ts

```diff
@@ -61,6 +61,8 @@ export class LayoutView extends View {
 
   private numOfPagesPerSpread: number = 0;
 
+  private loadIframesAsync: boolean = true;
```

Comment:
I suggest a better name for this flag, like `enableNonSequencialIframeLoading`. We are loading iframe as async. I think here is if iframes are loaded sequentially or not.

Reply:
Done